### ISSUE TITLE
If `n == 0` it still replaces one pipe

### ISSUE THREAD

**Issue by scheidan** [scheidan/WaMaSim#4]
Thanks for reporting Lukas!

The implementation of two strategies is wrong: 
- `replace.n.highest.risk`
- `replace.n.oldest`
 
If `n` is set to `0` it still replaces one pipe .
Mostleikely because of:
```R
idx <- order(inv$time.construction + as.numeric(!inv$in.service)*1E10)[1:(min(n, n.in.service))]
```
This causes some problems when running the simulation automatically and setting the values for n between 0 and 10 for example. It is not directly possible to then have a scenario where no pipes get replaced even if n is set to 0.

**Comment by ChristianF88** [comment]
I fixed it. Should be fine now.

**Comment by scheidan** [comment]
closed with d5533d71b4680810d8dc896a6df9b740eafbe896 .
